### PR TITLE
WatchKit support

### PR DESCRIPTION
**Pull request by kcharwood**
Including AFNetworking in a WatchKit 2 project results in a SystemConfiguration.h file not found error. 

I am seeing this in Xcode 7 / WatchOS 2 (beta 3)

### CODE REVIEW

File: Example/AFNetworking watchOS Extension/Assets.xcassets/README__ignoredByTemplate__ [hunk 1]
```diff
@@ -0,0 +1 @@
+Did you know that git does not support storing empty directories?
```
Comment (cnoon) [inline Example/AFNetworking watchOS Extension/Assets.xcassets/README__ignoredByTemplate__]: Should this be committed? Where did this file even come from?

File: Example/AFNetworking watchOS Extension/InterfaceController.m [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+- (void)didDeactivate {
+    // This method is called when watch view controller is no longer visible
+    [super didDeactivate];
+}
+
+@end
+
+
```
Comment (cnoon) [inline Example/AFNetworking watchOS Extension/InterfaceController.m]: Newline cleanup at the top and bottom.

File: Example/AFNetworking watchOS Extension/Assets.xcassets/README__ignoredByTemplate__ (hunk 1 above)
Comment (kcharwood) [inline Example/AFNetworking watchOS Extension/Assets.xcassets/README__ignoredByTemplate__]: I'm guessing xcode generated it when I added a watch target.

**Comment by cnoon** [comment]
My only other thing @kcharwood would be whether the project has actually been updated with the new deployment targets in the podspec. It would be a bummer to throw a bunch of compiler warnings for everyone that picks up the new release if there are any new deprecation warnings. Have you had a chance to see if that causes any problems?

**Comment by XenonChau** [comment]
```objective-c
- (NSURLRequest *)requestBySerializingRequest:(NSURLRequest *)request
                               withParameters:(id)parameters
                                        error:(NSError *__autoreleasing *)error
```

while "parameters" was converted into JSON or encrypted with AES.
the following assignment `query = AFQueryStringFromParametersWithEncoding(parameters, self.stringEncoding);` will cause an issue.

I don't know how many kinds of parameters. I'm using `[parameters isKindOfClass[NSDictionary class]] || [parameters isKindOfClass[NSArray class]] ` to avoid these two situations.

Is there any else?

**Pull request merged**